sky/packages/sky/lib/src/widgets/dismissable.dart

```diff
@@ -160,8 +222,8 @@ class Dismissable extends StatefulComponent {
 
   Widget build() {
     if (_resizePerformance != null) {
-      AnimatedValue<double> dismissHeight = new AnimatedValue<double>(
-        _size.height,
+      AnimatedValue<double> dismissSize = new AnimatedValue<double>(
```

Comment:
I'm not sure if I've improved things but. I called the variable squashAxisExtent since its the definitive parameter for a SquashTransition.